src/site/antora/modules/ROOT/pages/log4j-jul.adoc

```diff
@@ -15,116 +15,175 @@ Licensed to the Apache Software Foundation (ASF) under one or more
-xref:manual/systemproperties.adoc#log4j.jul.levelConverter[`log4j.jul.levelConverter`]
-must be set to your custom class name.
-Using the default `LevelConverter` implementation, custom logging levels are mapped to whatever the current level of the `Logger` being logged to is using.
-
-[#default-level-conversions]
-=== Default Level Conversions
-
-.JUL to Log4j level conversion
-[%header]
-|===
-| Java Level | Log4j Level
+https://docs.oracle.com/javase/{java-target-version}/docs/api/java/util/logging/Handler.html[`java.util.logging.Handler`]
+classes is **not** supported.
+Custom handlers should be replaced with the appropriate
+xref:manual/appenders.adoc[Log4j Appender].
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#OFF[`OFF`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#OFF[`OFF`]
+[#bridge-handler]
+== Using `Log4jBridgeHandler`
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#SEVERE[`SEVERE`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#ERROR[`ERROR`]
+.Are you a Spring Boot user?
+[TIP]
+====
+Spring Boot will automatically configure `Log4jBridgeHandler`.
+====
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#WARNING[`WARNING`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#WARN[`WARN`]
+If setting the `java.util.logging.manager` system property is not possible, the JUL-to-Log4j bridge offers an implementation of JUL's
+https://docs.oracle.com/javase/{java-target-version}/docs/api/java/util/logging/Handler.html[`Handler`]
+abstract class, which redirects all log events to Log4j Core:
+`org.apache.logging.log4j.jul.Log4jBridgeHandler`.
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#OFF[`OFF`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#INFO[`INFO`]
+[WARNING]
+====
+The `Log4jBridgeHandler` requires Log4j Core as logging implementation and will fail with other Log4j API implementations.
+====
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#CONFIG[`CONFIG`]
-| custom `CONFIG` level
+In order to use `Log4jBridgeHandler` you can either:
 
-| https://docs.oracle.com/en/java/javase/{java-target-version}/docs/api/java.logging/java/util/logging/Level.html#FINE[`FINE`]
-| {log4j2-url}/javadoc/log4j-api/org/apache/logging/log4j/Level.html#DEBUG[`DEBUG`]
+* modify the default JUL configuration file `logging.properties` to only contain:
++
+[source,properties]
+----
+# Set Log4jBridgeHandler as only handler for all JUL loggers
+handlers = org.apache.logging.log4j.jul.Log4jBridgeHandler
+----
++
+See the
+https://docs.oracle.com/javase/8/docs/api/java/util/logging/LogManager.html[JRE documentation]
```

Comment:
Shall we replace `8` with `{java-target-version}` instead? (There are some more places for such a change in this PR.)

Reply:
Right, yes, we can, but Java 9+ follow a different URI pattern (there are modules in play). I'll fix it.

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/2935/commits/ef72bf153195ba6bdfd3a7862b439578c549a2f2